src/test/java/akv/AkvIntegrationTestCase.java

```diff
@@ -0,0 +1,222 @@
+package akv;
+
+import com.avioconsulting.mule.connector.akv.provider.api.client.model.Certificate;
+import com.avioconsulting.mule.connector.akv.provider.api.client.model.Key;
+import com.avioconsulting.mule.connector.akv.provider.api.client.model.Secret;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.Context;
+import com.azure.core.util.polling.LongRunningOperationStatus;
+import com.azure.core.util.polling.PollResponse;
+import com.azure.core.util.polling.SyncPoller;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.security.keyvault.certificates.CertificateClient;
+import com.azure.security.keyvault.certificates.CertificateClientBuilder;
+import com.azure.security.keyvault.certificates.models.*;
+import com.azure.security.keyvault.keys.KeyClient;
+import com.azure.security.keyvault.keys.KeyClientBuilder;
+
+import com.azure.security.keyvault.keys.models.CreateRsaKeyOptions;
+import com.azure.security.keyvault.keys.models.DeletedKey;
+import com.azure.security.keyvault.keys.models.KeyVaultKey;
+import com.azure.security.keyvault.secrets.SecretClient;
+import com.azure.security.keyvault.secrets.SecretClientBuilder;
+import com.azure.security.keyvault.secrets.models.DeletedSecret;
+import com.azure.security.keyvault.secrets.models.KeyVaultSecret;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mule.functional.junit4.MuleArtifactFunctionalTestCase;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+
+public class AkvIntegrationTestCase extends MuleArtifactFunctionalTestCase {
+    private static String keyName;
+    private static String secretName;
+    private static String secretValue;
+    private static String certificateName;
+
+    @Override
+    protected String getConfigFile() {
+        return "integration-test-mule-config.xml";
+    }
+
+    private static void setGlobals(){
+        //System.setProperty("AZURE_CLIENT_ID", "");
+        //System.setProperty("AZURE_CLIENT_SECRET", "");
+        //System.setProperty("AZURE_TENANT_ID", "");
+        System.setProperty("AKV_TEST_KEY_NAME", getKeyName());
```

Comment:
Normally environment variables use this format but java system properties are usually lower case with . notation such as akv.test.key.name.  We should follow those commonly used standards.

Reply:
made changes to accept lowercase dot notation